src/FrameWorkController_002.test.js

```diff
@@ -0,0 +1,50 @@
+import {
+  getControllerPluginData,
+  pluginActivate,
+  pluginDeactivate,
+  restartFramework,
+} from './commonMethods/commonFunctions'
+
+export default {
+  title: 'Framework Controller test 02',
+  description: 'Tests basic functionality of the controller module',
+  teardown: restartFramework,
```

Comment:
I'd suggest to remove this. This was an artifact of a crashing WPEFramework, no need to do this anymore. WPEFramework should continue to function and not need a reboot after this test.